content/docs/command-reference/init.md

```diff
@@ -102,18 +94,37 @@ project-A
 └── project-B
     ├── .dvc
     ├── data-B.dvc
-    │   ...
+    ...
 ```
 
-Nothing changes for the `project-B`. But for any DVC command being run in the
-`project-A` ignores the whole directory `project-B/`, meaning for example:
+Nothing changes for inner `project-B`. But any DVC command being run in outer
+`project-A` ignores the nested directory `project-B/`. For example, `dvc pull`
+(run in `project-A/`) wouldn't download data for the `data-B.dvc` file.
+
+### Nested DVC repositories
```

Comment:
I guess confusion comes from the our usage of the DVC project term. DVC project to me is a generic term - it can be Git repo or not. So the whole section above should apply to "DVC repos" (which I would avoid using).

Reply:
Yes, a DVC repo is a kind of DVC project — we use these terms consistently (only call it "repo" when the Git context matters). In the monorepo pattern (--subdir), there's sub-projects in a single Git repo, so I don't call those DVC repos (that would be a little confusing). But anyway...

I'm working on merging this with the previous section as mentioned in https://github.com/iterative/dvc.org/pull/1615#discussion_r463272826, I'll get back to you... ⏳

Reply:
Merged in 1f77e8fe. But maybe it should be an Example instead of this long explanation with 2 code blocks?